crates/flox-rust-sdk/src/models/publish.rs

```diff
@@ -390,17 +408,19 @@ mod tests {
         };
 
         let narinfos = publish
-            .get_binary_cache_metadata(SubstituterUrl::parse("file:///nix/store").unwrap())
+            .get_binary_cache_metadata(None)
```

Comment:
```suggestion
            .get_binary_cache_metadata(SubstituterUrl::parse("file:///nix/store").unwrap())
```

Reply:
The problem with doing it that way is None is different than `/nix/store` - if you pass --store /nix/store to path-info, it fails:
```
~/flox builtfilter > nix path-info $(which flox) --json
[{"deriver":"/nix/store/4f6yn3nrvgv1wbdmpbzzwhbxxiis31zj-flox-prerelease-0.2.4-r552.drv","narHash":"sha256-4QtpkIahLNFz6du4Z8BtAtaP6NqQIefWG3ODjA8yc8A=","narSize":13519944,"path":"/nix/store/07j11vazqvhwhfyvr97rnzjyndkj3bz9-flox-prerelease-0.2.4-r552","references":["/nix/store/07j11vazqvhwhfyvr97rnzjyndkj3bz9-flox-prerelease-0.2.4-r552","/nix/store/5bks4hbp3pyvf13k3ifk8xm9mhk2q8zy-openssl-3.0.9","/nix/store/6nbhrl276fjp40j782grcl31z5v9lzwq-zlib-1.2.13","/nix/store/6zk5cb4f1wlwvbf0p5xqmshw5yd41g6y-nss-cacert-3.90","/nix/store/7hlf906gwwn8b5av5hi65dqxkh6wx1if-adv_cmds-119-locale","/nix/store/86w80ihrm09zchsxxhqqh77ic165yh5v-libssh2-1.11.0","/nix/store/a5allwqr652qgs5dacm0vmfdy16jrx3x-catalog-ingest","/nix/store/d9di9cna6c8k8szfcl3p4sgrkkscjc2s-nix_bash_completion.sh","/nix/store/fqammp4dxb721d0zzjgaf8spwjrnwgcv-libiconv-50","/nix/store/j8wda87g7jn45q7pzfdyvbziljh7j9hp-git-minimal-2.41.0","/nix/store/kl8i709anpj3hvq7qpnaqz7mygs7196s-resolver","/nix/store/m7hhwa8s6012aga4rhw4xhps7527frz2-nix_zsh_completion.sh","/nix/store/nzsnsxhc2grpc5628icn8drd0i03kijk-apple-framework-CoreFoundation-11.0.0","/nix/store/x6hcyhaibk4rajfkf0r5kf9q34qs9gcz-flox-bash-0.2.4-r552"],"registrationTime":1689801350,"signatures":["flox-store-public-0:WQpGKgqm2Aa4EycbvwKXenM7T5DCIhX8vGYz4KaFUUCr324PHGRl6GAsvvFHpnE87WSoisAST8dfCt7cInUKAQ=="],"valid":true}]%
~/flox builtfilter > nix path-info $(which flox) --json --store /nix/store
error: creating directory '/nix/store/nix': Permission denied
~/flox builtfilter > nix path-info $(which flox) --json --store /
error: could not set permissions on '//nix/var/nix/profiles/per-user' to 755: Operation not permitted
```

Reply:
So should I leave as is and ignore all suggestions?